build-tools/xaprepare/xaprepare/ConfigAndData/Runtimes.cs

```diff
@@ -10,21 +10,49 @@ partial class Runtimes
 		public readonly List<Runtime> Items = new List <Runtime> {
 			new MonoJitRuntime (
 				abiName: AbiNames.TargetJit.AndroidArmV7a,
+				interpreter: false,
 				enabledCheck: (Context ctx) => ctx.IsTargetJitAbiEnabled (AbiNames.TargetJit.AndroidArmV7a)
 			),
 
+			new MonoJitRuntime (
+				abiName: AbiNames.TargetJit.AndroidArmV7a,
+				interpreter: true,
+				enabledCheck: (Context ctx) => ctx.IsTargetJitAbiEnabled (AbiNames.TargetJit.AndroidArmV7a)
+			),
+
+			new MonoJitRuntime (
+				abiName: AbiNames.TargetJit.AndroidArmV8a,
+				interpreter: false,
+				enabledCheck: (Context ctx) => ctx.IsTargetJitAbiEnabled (AbiNames.TargetJit.AndroidArmV8a)
+			),
+
 			new MonoJitRuntime (
 				abiName: AbiNames.TargetJit.AndroidArmV8a,
+				interpreter: true,
 				enabledCheck: (Context ctx) => ctx.IsTargetJitAbiEnabled (AbiNames.TargetJit.AndroidArmV8a)
 			),
 
 			new MonoJitRuntime (
 				abiName: AbiNames.TargetJit.AndroidX86,
+				interpreter: false,
 				enabledCheck: (Context ctx) => ctx.IsTargetJitAbiEnabled (AbiNames.TargetJit.AndroidX86)
 			),
 
+			new MonoJitRuntime (
+				abiName: AbiNames.TargetJit.AndroidX86,
+				interpreter: true,
+				enabledCheck: (Context ctx) => ctx.IsTargetJitAbiEnabled (AbiNames.TargetJit.AndroidX86)
+			),
```

Comment:
Alas we do not support the interpreter on `x86`.

Is this fairly common? Or does everyone use `x86_64` anyway as emulator?

Reply:
We actually use `x86` emulators on CI...

Reply:
I meant with "everyone" the average Xamarin.Android developer 🙂

Reply:
Developers will most definitely run emulators using either `x86` or `x86_64` images, but I don't know which one of those is more popular. Both are accelerated and `x86_64` can run `x86` apps, so I'd **hope** that this one's more popular, but that's just my hope not supported by any data :)

Reply:
ok, I created https://github.com/mono/mono/issues/19669

Reply:
I think most devs use x86 right now as Visual Studio suggests it by default.

Reply:
@lewurm x86 is like 90% with x86 in the < 10% range. ARM even lower here.

Reply:
ABI/CPU - 

88/10/3/2% for x86/x86_64/armeabi-v7a/arm64-v8a on Mac. 

90/7/1/1% for x86/x86_64/armeabi-v7a/arm64-v8a on Windows.